src/compiler/python_generator.cc

```diff
@@ -777,7 +777,8 @@ pair<bool, grpc::string> PrivateGenerator::GetGrpcServices() {
     if (generate_in_pb2_grpc) {
       out->Print(
           "# Generated by the gRPC Python protocol compiler plugin. "
-          "DO NOT EDIT!\n");
+          "DO NOT EDIT!\n\"\"\""
+          "Protocol buffer generated client and server classes.\"\"\"\n");
```

Comment:
The current wording could be interpreted as "Classes generated *by* protocol buffers." Perhaps "Client and server classes corresponding to protobuf-defined services."?

Reply:
@gnossen I got inspiration from [here](https://grpc.io/docs/tutorials/basic/python/) but I think your phrasing is better. Can you please suggest an equivalent in the twin PR [here](https://github.com/protocolbuffers/protobuf/pull/7528)? I will change both then.

Reply:
As for your other PR, things are a bit trickier on the `_pb2.py` side. The contents of those files can't be summed up as crisply as "classes". Maybe just "Generated protocol buffer code."

Reply:
Changed it to the suggested wording @gnossen. Thanks.